### DISCUSSION TITLE
github student developer pack issue

### DISCUSSION THREAD

**Discussion by Imadali595** [community/community#19207] [GitHub Education] [answered]
hello guy,
 I want to sign up for the student developer pack. I'm currently doing my undergraduate from the University of Peshawar and the email id issued to me by my university is std102005@uop.edu.pk  but when I'm trying to apply they show me that this email domain belongs to the university of people now what I will do. can GitHub resolve this issue? here are the snaps
![xx](https://user-images.githubusercontent.com/74678609/175124049-384155ef-02a8-4818-976c-cf16cebec7c7.PNG)
![ss](https://user-images.githubusercontent.com/74678609/175124063-1335285c-3133-49ca-b9cd-179aa4d99219.PNG)

**Comment by rilysh** [comment]
GitHub said that your mail address must ends with `upesh.edu` or `upesh.edu.pk`, this seems kind of wrong to me, it's recommended to contact with the University and GitHub to resolve the issue, in discussion, less chance that anyone have same issue with same University mail address.

GitHub support: https://support.github.com/
And describe the current situation also if you believe it's a mistake make sure you've contact with University too.

**Reply by Imadali595** [reply]
my university email domain ends with uop.edu.pk
so what i am supposed to do

**Comment by melvincwng** [comment, marked as answer]
@Imadali595  kindly raise a ticket on [Github Support](https://support.github.com/contact/account)  to resolve account-related issues.

Hope this clarifies.

**Comment by Zhaoyang-Ma** [comment]
I am a college Student in college. I cannot join the GitHub Student Developer Pack if I have connected my edu.cn email and uploaded my Student ID photo with registration date or screenshots of Student status for four times.What should I do to  join the GitHub Student Developer Pack

**Comment by conngriff** [comment]
@Conngriff22 kindly raise a ticket on [Github Support](https://support.github.com/contact/account) to resolve account-related issues.

**Comment by conngriff** [comment]
#28581
